### PR TITLE
Cron jobs

### PR DESCRIPTION
**Pull request by vacekj**

### CODE REVIEW

**Comment by hmrtn** [comment]
Can you squash the commits and resolve the conflicts before approval?

**Comment by vacekj** [comment]
This is currently lacking tests, but I'm holding off on those because the changes that are coming to the API package might significantly impact how we do testing - trying to justify an exception here.

**Review by thelostone-mc — APPROVED** [review]
codewise this makes sense but curious on why we can't add tests atm

File: generate_commit_message.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# Check if the user confirmed
+if [ "$CONFIRM" == "y" ]; then
+  echo "Creating PR"
+else
+  echo "Aborting."
+  exit 1
+fi
+
+# Create a pull request
+gh pr create -H main -B release -t "Production release $CURRENT_DATE by $AUTHOR" -b "$COMMIT_MESSAGE"
```
Comment (thelostone-mc) [inline generate_commit_message.sh:70]: ^ could we throw this as a separate PR ?

**Comment by melissa-neira** [comment]
defer to engineering 🦕

**Comment by nfrgosselin** [comment]
Defer to engineering 🔧

**Pull request merged**